etc/telegraf/telegraf.toml.j2

```diff
@@ -335,3 +335,8 @@
 
 ####################################################################################
 {% endif %}
+
+[[outputs.exec]]
+  command = ["python3", "/home/vcap/app/buildpack/telemetry/appdynamics_telegraf_output.py"]
+  timeout = "10s"
+  data_format = "json"
```

Comment:
- Let's enable the output block only when Appdynamics is enabled
- Do add a comment to describe the use case here.

Reply:
Done